### PR TITLE
Small fixups of util/process_docs.pl

### PR DESCRIPTION
**Pull request by levitte**
##### Checklist

- [x] CLA is signed

##### Description of change

- the pod path hadn't been changed with the directory layout change
- apparently, pod2html doesn't add ".html" at the end of links, making
  them useless, so we need to fix that

With thanks for the report to Michel <michel.sales@free.fr>

### CODE REVIEW

File: util/process_docs.pl [hunk 1]
```diff
@@ -90,7 +90,7 @@
         my $suffix = { man  => ".$podinfo{section}",
                        html => ".html" } -> {$options{type}};
         my $generate = { man  => "pod2man --name=$name --section=$podinfo{section} --center=OpenSSL --release=$config{version} \"$podpath\"",
-                         html => "pod2html \"--podroot=$options{sourcedir}\" --htmldir=$updir --podpath=apps:crypto:ssl \"--infile=$podpath\" \"--title=$podname\""
+                         html => "pod2html \"--podroot=$options{sourcedir}\" --htmldir=$updir --podpath=man1:man3:mn5:man7 \"--infile=$podpath\" \"--title=$podname\""
```
Comment (kaduk) [inline util/process_docs.pl]: man5 takes an 'a'.

**Comment by levitte** [comment]
Separate PR for 1.1.0 in #1897

File: util/process_docs.pl (hunk 1 above)
Comment (levitte) [inline util/process_docs.pl]: Oops
Will fix

**Review by richsalz — APPROVED** [review]

**Comment by levitte** [comment]
Merged

**Pull request closed without merging**